apps/prairielearn/python/prairielearn.py

```diff
@@ -766,7 +789,12 @@ def numpy_to_matlab(np_object, ndigits=2, wtype="f"):
         return matrix_str
 
 
-def string_from_numpy(A, language="python", presentation_type="f", digits=2):
+def string_from_numpy(
+    A: np.ndarray | NumPyScalarType,
+    language="python",
```

Comment:
The language can be a literal made from a list.

Reply:
Fixed in 5027838266fba11cc95373b6954f1c928a030c78.